packages/block-library/src/cover/style.scss

```diff
@@ -1,6 +1,6 @@
-.wp-block-cover-image,
```

Comment:
The block used to be called "Cover Image". Do we need a deprecation handler for people who inserted the block when it was called that, and who rely in this CSS? 

Keep in mind someone might have installed the plugin midways, and created a bunch of reusable blocks of this. These might break?